### ISSUE TITLE
Handle unboxed tuples caused by `-funbox-strict-fields`

### ISSUE THREAD

**Issue by zliu41** [con-kitty/categorifier#46]
Handle unboxed tuples caused by `-funbox-strict-fields`

(Extricated from https://kitty-hawk.atlassian.net/browse/SW-3652)

**Comment by sellout** [comment]
Should we mention this in the README, and perhaps set `-fno-unbox-strict-fields` in the `DynFlagsPlugin` (assuming I ever get around to adding it).

**Comment by zliu41** [comment]
Yeah, I can add it to the README when I get around to finding a suitable example. The one we encountered is

```haskell
Linear.V3.$wcross :: forall a. Num a => a -> a -> a -> a -> a -> a -> (# a, a, a #)
```

And of course there's no `HasRep (# C Double, C Double, C Double #)`.
